apps/wallet/src/ui/app/SentryRpcClient.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
does it make sense to maintain separate but identical `SentryRpcClient` classes for the wallet and explorer? or might we have slightly different constraints based on the use case?

Reply:
Nope! It doesn't at all 😅 
We've been meaning to create a shared location for code like this for a while (`@mysten/core` is intended to be _roughly_ this) we just haven't integrated it with the projects yet.